windows/src/developer/kmcmpdll/kmcmpdll.h

```diff
@@ -6,6 +6,8 @@
 BOOL AddCompileString(LPSTR buf);
 BOOL AddCompileMessage(DWORD msg);
 
+// TODO: These macros can return FALSE in functions that expect a DWORD CERR_x
+//       return value type. This is just plain wrong!
```

Comment:
So is there a plan to address this?  Just having this comment warns the developer for future use, but what about for current use cases?

Reply:
I haven't opened an issue on this. I think we should, at some point, revisit the compiler error messages, but it's lowpri so for now I just want to leave a TODO in the code. (We are not seeing egregious failures due to this, so in some ways it's a curiosity.)

From my perspective, a TODO in the code is something that we can turn into an issue for resolution as/when we have time -- without overwhelming our issue lists. I try not to create new TODOs for new code but add them to existing code when maintaining it -- particularly when in a beta cycle, TODO may be a preference over a fix.

Reply:
No worries. Definitely don't want the extra work in the beta cycle.  Are we really ever going to go through the TODO list. A quick search shows there are 3700 + TODOs. A number of those are blank from the title block but still could safely say there in the 100s.

Reply:
> Are we really ever going to go through the TODO list. A quick search shows there are 3700 + TODOs. A number of those are blank from the title block but still could safely say there in the 100s.

This is a job for an intern!